tests/integration/test_optislang/test_optislang_integration.py

```diff
@@ -36,69 +39,80 @@ def test_simple_solve(load_mixing_elbow_param_case_dat):
     solver_session = load_mixing_elbow_param_case_dat
     assert solver_session.health_check.is_serving
     case_path = examples.path("elbow_param.cas.h5")
-    solver_session.tui.file.read_case_data(case_path)
+    solver_session.settings.file.read_case_data(file_name=case_path)
 
     # Step 3: Get input and output parameters and create a dictionary
-    from ansys.fluent.core.services.scheme_eval import Symbol
+    reader = CaseFile(case_file_name=case_path)
```

Comment:
Why didn't we use the live session to get the input/output parameters?

Reply:
@mkundu1 We are quering input/output parameters from CaseFile during parametrization and from session during the evaluation. May be its a good idea to test both the scenarios. I have noticed sometimes that the values returned from session are of different unit system than the values returned from CaseFile.